modules/imgcodecs/test/test_common.cpp

```diff
@@ -12,7 +12,7 @@ Mat generateTestImageBGR_()
     Size sz(640, 480);
     Mat result(sz, CV_8UC3, Scalar::all(0));
 
-    const string fname = cvtest::findDataFile("../cv/shared/baboon.png");
+    const string fname = cvtest::findDataFile("../python/images/baboon.bmp");
```

Comment:
I believe using `cv/shared` is less confusing than using image from `python/images`. Maybe we can copy that BMP to `cv/shared`?